docs/examples/writeData.rst

```diff
@@ -0,0 +1,125 @@
+.. _exampleWriteData:
+
+Writing Data
+============
+
+This section demonstrates how the :code:`isx` package can be used to write data to Inscopix files.
+Refer to :ref:`this table <fileTypes>` for reference on the Inscopix file types and write support.
+
+.. note::
+   
+   The following sections assume the :code:`isx` package has been imported, i.e., :code:`import isx`
+
+Microscope Movies
+-----------------
+
+The `Microscope Movie` file type can be written using the :code:`isx.Movie` class. 
+
+.. code-block:: python
+
+    # Open the movie for writing
+    movie = isx.Movie.write('movie.isxd', timing, spacing, numpy.float32)
```

Comment:
I'm not sure how much of this is auto generated, but here the timing and spacing are not very well, explained, maybe you can just quickly link to where they're defined in `reference.html`?